code/datums/abilities/carnival_cant.dm

```diff
@@ -0,0 +1,37 @@
+/datum/abilityHolder/carnival
+    usesPoints = 0
+    regenRate = 0
+    tabName = "Carnival"
+
+/datum/targetable/carnival/cant //defaults to clown version
+    name = "Carnival Cant"
+    desc = "A secret language that only entertainers know! This one in particular is a form of sign language!"
+    icon_state = "cant-clown"
+    targeted = 0
+    cooldown = 0
+    pointCost = 0
```

Comment:
This will keep the base type out of `concrete_typesof` and I suppose we might as well just slide up some vars while we're at it. Additionally sets our preferred holder type for carnival abilities

```suggestion
ABSTRACT_TYPE(/datum/targetable/carnival)
/datum/targetable/carnival
	cooldown = 0
	last_cast = 0
	pointCost = 0
	preferred_holder_type = /datum/abilityHolder/carnival

/datum/targetable/carnival/cant //defaults to clown version
    name = "Carnival Cant"
    desc = "A secret language that only entertainers know! This one in particular is a form of sign language!"
    icon_state = "cant-clown"
    targeted = 0

```

Reply:
oooh thank you